projects/plugins/jetpack/extensions/shared/memberships/settings.scss

```diff
@@ -15,6 +15,7 @@
 .jetpack-newsletter-access-radio-buttons {
 	label {
 		text-wrap: balance;
+		max-width: 100%;
```

Comment:
I like this much better than no-wrap, which can cause unexpected issues with longer translated texts. 👍

Reply:
@chrisfromthelc Was supposed to tinker with that approach as well: p1708017082462599/1688455659.702199-slack-C01B6KEJ5GE
Not sure how it went

Reply:
@pkuliga was looking into these lines as well recently, and noted that the problem seemed to be too narrow labels, not the lack of space in the sidebar per-se.